etl/tests/test_dwelling.py

```diff
@@ -564,8 +565,8 @@ def test_from_row(self, sample_input_d: reader.InputData) -> None:
             ventilations=[
                 ventilation.Ventilation(
                     ventilation_type=ventilation.VentilationType.NOT_ENERGY_STAR_NOT_INSTITUTE_CERTIFIED,
-                    air_flow_rate=220,
-                    efficiency=55,
+                    air_flow_rate=220.0,
+                    efficiency=55.0,
```

Comment:
Technically `==`, but shows up as a difference in the output string when other things are different.